remoteappmanager/rest/rest_handler.py

```diff
@@ -25,6 +26,44 @@ def get_resource_handler_or_404(self, collection_name):
         except KeyError:
             raise web.HTTPError(httpstatus.NOT_FOUND)
 
+    def write_error(self, status_code, **kwargs):
+        """Provides appropriate payload to the response in case of error.
+        """
+        exc_info = kwargs.get("exc_info")
+
+        if exc_info is None:
+            self.clear_header('Content-Type')
```

Comment:
'Content-Type' is by default not set unless you overload `set_default_header`.  Before calling `write_error`, `send_error` reset the header using `set_default_header`.  Why do we need to clear 'content-type' here? (no effect if `set_default_header` is passed, override `set_default_header` if it was defined for 'content-type')